core/src/providers/tiktoken/tiktoken.rs

```diff
@@ -506,6 +553,10 @@ impl CoreBPE {
         self._encode_native(text, &allowed_special).0
     }
 
+    pub fn tokenize_with_matching_string(&self, text: &String) -> Vec<(usize, String)> {
```

Comment:
Let's add a comment to say it's our implementation

Reply:
Let's call it `tokenize`?

Reply:
Done!

Reply:
You forget this one?